inc/TRestAxionField.h

```diff
@@ -57,6 +57,14 @@ class TRestAxionField : public TObject {
     Double_t AxionAbsorptionProbability(Double_t Bmag, Double_t Lcoh, Double_t Ea, Double_t ma,
                                         Double_t mg = 0, Double_t absLength = 0);
 
+    Double_t GammaTransmissionFWHM(Double_t m_a = 0, Double_t Ea = 4, Double_t Bmag = 2.5,
+                                   Double_t Lcoh = 10000, Double_t mg = 0, Double_t step = 0.00001,
+                                   int n = 10000);
+
+    std::pair<std::vector<Double_t>, std::vector<Double_t>> GetMassDensityScanning(std::string gasName = "He",
```

Comment:
I am not sure this is the best return format. Would `std::vector < std::pair < Double_t, Double_t> >` be more convenient? Then, when you get an element of the vector that element is a pair that contains the mass and its corresponding density. Perhaps @lobis got some comments on this.

Reply:
Yes, I think that `std::vector < std::pair < Double_t, Double_t> >` is definitely better, as the two values are related and they always appear as pairs.